catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/CatalogPostgreSqlIT.java

```diff
@@ -747,6 +776,55 @@ void testColumnDefaultValue() {
         Literals.integerLiteral(1000), createdTable.columns()[5].defaultValue());
   }
 
+  @Test
+  void testUpdateColumnDefaultValue() {
+    Column[] columns = columnsWithDefaultValue();
+    Table table =
+        catalog
+            .asTableCatalog()
+            .createTable(
+                NameIdentifier.of(metalakeName, catalogName, schemaName, tableName),
+                columns,
+                table_comment,
+                createProperties());
+
+    catalog
+        .asTableCatalog()
+        .alterTable(
+            NameIdentifier.of(metalakeName, catalogName, schemaName, tableName),
+            TableChange.updateColumnDefaultValue(
+                new String[] {columns[0].name()},
+                toFunctionArg(Literals.of("1.234", Types.FloatType.get()))),
```

Comment:
It's unnecessary to use `toFunctionArg`

Reply:
The [Serde of ColumnDefaultValue](https://github.com/datastrato/gravitino/blob/main/common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java#L1059) will cast the `LiteralImpl` to `FunctionArg`. If `toFunctionArg` is not used, an exception will be encountered.

Reply:
You should move `toFunctionArg` to [there](https://github.com/datastrato/gravitino/pull/2673/files#diff-49ca60933fb4234ec2833daacdf6ea31cf77579c5bc46e7aecdfd0adeacadadbR204)